### PR TITLE
docs: add registration function step to Amiibo tutorial

### PR DESCRIPTION
**Pull request by angethuy**
### Description
This PR adds an additional step to the Amiibo tutorial's documentation. It guides the user in updating the scaffolded index.tsx file by providing a sample file and highlighting the changed lines.  

### Motivation
To successfully register the Amiibo workflow, we also need to update the registration function to reflect the component name changed in previous tutorial steps.

### Testing Performed
Manually followed tutorial steps and confirmed that the workflow does not get registered until the registration function is also updated.

### CODE REVIEW

File: docs/development/feature.md [hunk 1]
```diff
@@ -549,7 +549,39 @@ Here we define the data layout structure and pass it to the wizard along with th
 
 ##### Configuration
 
-The final step is registering this new Amiibo workflow with the Clutch app. Open your `clutch.config.js` file and add the following:
+The final step is registering this new Amiibo workflow with the Clutch app. First update the component name by replacing the scaffolded default in our workflow's registration function.
```
Comment (dschaller) [inline docs/development/feature.md:552]:
```suggestion
The final step is registering this new Amiibo workflow with the Clutch app. First update the component name by replacing the scaffolded default (`HelloWorld`) in our workflow's registration function.
```

**Comment by dschaller** [comment]
Awesome! Thanks for the contribution :)

File: docs/development/feature.md [hunk 2]
```diff
@@ -549,7 +549,39 @@ Here we define the data layout structure and pass it to the wizard along with th
...
+// highlight-next-line
+import Amiibo from "./hello-world";
+
+export interface WorkflowProps extends BaseWorkflowProps {}
+
+const register = (): WorkflowConfiguration => {
+  return {
+    developer: {
+      name: "Name McName",
+      contactUrl: "mailto:foo@foo-email.com>",
```
Comment (danielhochman) [inline docs/development/feature.md:565]:
```suggestion
      contactUrl: "mailto:foo@foo-email.com",
```

**Review by danielhochman — COMMENTED** [review]
great addition, one small typo

**Review by danielhochman — APPROVED** [review]
:balloon:

**Pull request merged**